README.md

```diff
@@ -10,9 +10,8 @@ pip install --upgrade git+git://github.com/keboola/python-transformation-v2.git
 
 ### Dev
 - clone the repo
-- `export BASE_IMAGE=quay.io/keboola/docker-custom-python:latest` or provide the tag you want, say `2.2.0`
-- `./create_dockerfile.sh`
-- `docker-compose build`
+- `docker-compose build --build-arg BASE_IMAGE=quay.io/keboola/docker-custom-python:latest`
+  or provide the tag you want, say `2.2.0`
```

Comment:
from the Dockerfile I think this should rather be `BASE_IMAGE_NAME=uay.io/keboola/docker-custom-python` and `BASE_IMAGE_TAG=latest`

Reply:
Oh, sure, thanks!